code/modules/mob/living/simple_animal/slime/powers.dm

```diff
@@ -99,6 +99,14 @@
 			return FALSE
 		to_chat(src, "<span class='warning'><i>Another slime is already feeding on this subject...</i></span>")
 		return FALSE
+
+	if(iscarbon(M))
+		var/mob/living/carbon/C = M
+		if((C.wear_suit && ((C.wear_suit.flags & THICKMATERIAL)) && ((C.head && ((C.head.flags & THICKMATERIAL))))))
```

Comment:
the amount of parenthesis make it a bit hard to read. Can just write it like this:
`suit && (suit & THICK) && head && (head & THICC)`

Reply:
Yea that is better, I'll make that change.